src/components/UserSelector.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export const UserSelector: React.FC = () => {
```

Comment:
Remove redundant and unused code from this task. Remove all redundant components